homeassistant/components/librehardwaremonitor/sensor.py

```diff
@@ -0,0 +1,108 @@
+"""Support for LibreHardwareMonitor Sensor Platform."""
+
+from __future__ import annotations
+
+from librehardwaremonitor_api.model import LibreHardwareMonitorSensorData
+
+from homeassistant.components.sensor import SensorEntity, SensorStateClass
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import LibreHardwareMonitorCoordinator
+from .const import DOMAIN
+from .coordinator import LibreHardwareMonitorConfigEntry
+
+# Coordinator is used to centralize the data updates
+PARALLEL_UPDATES = 0
+
+STATE_MIN_VALUE = "min_value"
+STATE_MAX_VALUE = "max_value"
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: LibreHardwareMonitorConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up the LibreHardwareMonitor platform."""
+    lhm_coordinator: LibreHardwareMonitorCoordinator = config_entry.runtime_data
+
+    sensor_entities = [
+        LibreHardwareMonitorSensor(lhm_coordinator, sensor_data)
+        for sensor_data in lhm_coordinator.data.sensor_data.values()
+    ]
+
+    async_add_entities(sensor_entities)
+
+
+class LibreHardwareMonitorSensor(
+    CoordinatorEntity[LibreHardwareMonitorCoordinator], SensorEntity
+):
+    """Sensor to display information from LibreHardwareMonitor."""
+
+    _attr_state_class = SensorStateClass.MEASUREMENT
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: LibreHardwareMonitorCoordinator,
+        sensor_data: LibreHardwareMonitorSensorData,
+    ) -> None:
+        """Initialize an LibreHardwareMonitor sensor."""
+        super().__init__(coordinator)
+
+        self._attr_name: str = sensor_data.name
+        self.value: str | None = sensor_data.value
+        self.attributes: dict[str, str] = {
+            STATE_MIN_VALUE: self._format_number_value(sensor_data.min),
+            STATE_MAX_VALUE: self._format_number_value(sensor_data.max),
+        }
+        self._unit_of_measurement: str | None = sensor_data.unit
+        self._attr_unique_id: str = f"lhm-{sensor_data.sensor_id}"
+
+        self._sensor_id: str = sensor_data.sensor_id
+
+        # Hardware device
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, sensor_data.device_name)},
+            name=sensor_data.device_name,
+            manufacturer="Hardware",
```

Comment:
who is the manufactuerer?

Reply:
It's not possible to get the manufacterer from LHM, I removed the "Hardware" placeholder.